commands/commands.go

```diff
@@ -57,6 +57,17 @@ func Exit(format string, args ...interface{}) {
 	os.Exit(2)
 }
 
+func ExitWithError(err error) {
+	if Debugging || lfs.IsFatalError(err) {
+		Panic(err, err.Error())
+	} else {
+		if inner := lfs.GetInnerError(err); inner != nil {
```

Comment:
No need because of [this](https://github.com/github/git-lfs/blob/3daf3d4a135a6e42f0e9f0b45cb60a2d922d27c5/lfs/errors.go#L296-L301). Good call though, I added a test confirming it.